source/connectors/kafka-sink-properties.txt

```diff
@@ -0,0 +1,239 @@
+   * - post.processor.chain
+     - list
+     - | A list of post-processor classes that process the data before saving it to MongoDB.
+       |
+       | **Default**: ``com.mongodb.kafka.connect.sink.processor.DocumentIdAdder``
+       | **Accepted Values**: A comma-separated list of fully qualified Java class names
+
+   * - rate.limiting.every.n
+     - int
+     - | Number of processed batches that should trigger the rate limit. A value of 0 means no rate limiting.
+       |
+       | **Default**: ``0``
+       | **Accepted Values**: An integer
+
+   * - rate.limiting.timeout
+     - int
+     - | How long (in milliseconds) to wait before continuing to process data once the rate limit is reached.
+       |
+       | **Default**: ``0``
+       | **Accepted Values**: An integer
+
+   * - value.projection.list
+     - string
+     - | A list of field names to include in the value projection.
+       |
+       | **Default**: ``""``
+       | **Accepted Values**: A comma-separated list of field names
+
+   * - value.projection.type
+     - string
+     - | The type of value projection to use.
+       |
+       | **Default**: ``none``
+       | **Accepted Values**: ``none``, ``blacklist``, or ``whitelist``
+
+   * - writemodel.strategy
+     - string
+     - | The class that specifies the ``WriteModel`` to use for :manual:`Bulk Writes </core/bulk-write-operations/index.html>`.
+       |
+       | **Default**: ``com.mongodb.kafka.connect.sink.writemodel.strategy.ReplaceOneDefaultStrategy``
+       | **Accepted Values**: A fully qualified Java class name of a class that implements ``WriteModelStrategy``
+
+   * - topic.override.<topicName>.<propertyName>
+     - string
+     - | Per-topic configurations that override the corresponding global and default property settings. For example: ``topic.override.foo.collection=bar`` instructs the sink connector to store data from the ``foo`` topic in the ``bar`` collection. For additional examples, see :ref:`topic-specific-configuration`.
+
+       .. note::
+
+          You can specify any configuration on a per-topic basis except
+          for ``connection.uri`` and ``topics``.
+
+       | **Default**:  ``""``
+       | **Accepted Values**: Accepted values specific to the overridden property
+
+.. _topic-specific-configuration:
+
+Topic-Specific Configuration Settings
+-------------------------------------
+
+The MongoDB Kafka Sink Connector can be configured to sink data from
+multiple topics. You can override global or default property settings with
+a topic-specific setting in the form of
+``topic.override.<topicName>.<propertyName>``.
+
+.. note::
+
+   The ``topics`` and ``connection.uri`` properties are global and
+   **cannot** be overridden.
+
+The following example demonstrates specifying topic-specific settings.
+
+Example: Override Connector Sink Settings on TopicA
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: properties
+
+   topic.override.topicA.collection=collectionA
```

Comment:
Anyway to bold the prefix? **`topic.override.topicA.`**`collection=collectionA` - if not no worries.

Reply:
I am not aware of any way to bold an arbitrary segment of text in a code-block since it follows syntax highlighting styles only. While it may not have the same effect, I can add an `emphasize-lines` directive to apply a yellow highlight to the first line if you think that is helpful. It would highlight the entire line, not just to `topicA`. Another idea is to add a linebreak. Let me know what you think works best.

Reply:
No problems lets just leave as it is